holidays/countries/united_states.py

```
@@ -249,7 +249,7 @@ def __init__(self, *args, **kwargs):
         kwargs.setdefault("observed_since", 1953)
         super().__init__(*args, **kwargs)
 
-    def _populate_common(self, include_federal: bool = False):
+    def _populate_common(self, *, include_federal: bool = False):
```

Ruff (0.14.0)

Missing return type annotation for private function `_populate_common`

Add return type annotation: `None`

(ANN202)